src/console/Command.hpp

```diff
@@ -0,0 +1,60 @@
+#ifndef CONSOLE_COMMAND_HPP
+#define CONSOLE_COMMAND_HPP
+
+#include "storm/Hash.hpp"
+#include <cstdint>
+
+#define EXEC_BUFFER_SIZE 8192
+#define CMD_BUFFER_SIZE 1024
+#define MAX_CMD_LENGTH 64
+#define HISTORY_DEPTH 32
+#define NOHELP nullptr
+
+enum CATEGORY {
+    DEBUG,
+    GRAPHICS,
+    CONSOLE,
+    COMBAT,
+    GAME,
+    DEFAULT,
+    NET,
+    SOUND,
+    GM,
+    NONE,
+    LAST
+};
+
+struct CONSOLECOMMAND : TSHashObject<CONSOLECOMMAND, HASHKEY_STRI> {
+    const char* command;
+    int32_t (*handler)(const char*, const char*);
+    const char* helpText;
+    CATEGORY category;
+};
```

Comment:
Do any of these members get initialized to default values at construction time? If so, we should add inline initializers to the member declarations. If you know where these objects are instantiated, there should be clues as to whether or not they're getting initialized at construction time.

Reply:
It doesn't look like they get initialized to default values at construction time to me.